committeeoversightapp/templates/delete.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="hearing-form card">
+  <h1>Delete Hearing Record</h1>
+  <div class="float-right confirm-delete">
+    <i>Are you sure you want to delete this hearing?</i>
+  </div><br />
+  <table class="table table-striped table-bordered" id="hearing-table">
+   <thead>
+   </thead>
+   <tbody>
+     <tr>
+       <td><b>Hearing Title</b></td>
+       <td>{{hearing.name | title}}</td>
+     </tr>
+     <tr>
+       <td><b>Hearing Date</b></td>
+       <td>{{hearing.start_date}}</td>
+     </tr>
+     <tr>
+       <td><b>Date Entered</b></td>
+       <td>{{hearing.created_at | date:"Y-m-d"}}</td>
+     </tr>
+     <tr>
+       <td><b>Committees</b></td>
+       <td>{{committees}}</td>
+     </tr>
+     <tr>
+       <td><b>Category</b></td>
+       <td>{{category_name}}</td>
+     </tr>
+   </tbody>
+   </table>
+
+   <form method="POST" class="form">
+     {% csrf_token %}
+     <button type="submit" class="btn btn-danger float-right" id="delete-hearing-button">
+       <i class="fas fa-times-circle"></i>&nbsp;
+       Delete hearing
+     </button>
+   </form>
```

Comment:
It might be nice to have a "Go back" button on this page, which directs the user to the list view.

Reply:
I like it